blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/Discovery.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.io;
+
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentFactory;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Discovery {
+    private Map<String, Object> annotation = new HashMap<String, Object>();
+    private final String metricName;
+    private final String tenantId;
+
+    public Discovery(String tenantId, String metricName) {
+        this.tenantId = tenantId;
+        this.metricName = metricName;
+    }
+
```

Comment:
This Discovery class is extracted from ElasticIO (where it was an inner class) so that the new EnumElasticIO class can also use it.